### DISCUSSION TITLE
Display showing build/version [DONE]

### DISCUSSION THREAD

**Discussion by DvV74** [probonopd/MiniDexed#739] [Ideas]
Is it possible to have the display show the build version/date at startup?

**Comment by probonopd** [comment]
Hello @DvV74 it'd certainly be doable but no one has implemented it so far. Ideally, we would show the short (7-digit) sha hash of the respective git command to easily find the matching source code.

**Reply by probonopd** [reply]
This will do it:

* https://github.com/probonopd/MiniDexed/pull/747/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R24-R25

The latest build from https://github.com/probonopd/MiniDexed/pull/747 already has it.
